airflow/api_connexion/openapi/v1.yaml

```
@@ -377,6 +377,7 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
+        - $ref: '#/components/parameters/OnlyActive'
```

```suggestion
       - name: only_active
          in: query
          schema:
            type: boolean
          required: false
          description: Only return DAGs which were seen on the last DagBag load.
```
Since `OnlyActive` is only used in this endpoint, It makes sense to have the parameters here instead of creating a component for it